lib/ansible/modules/cloud/vmware/vmware_host_facts.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Wei Gao <gaowei3@qq.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_host_facts
+short_description: Gathers facts about remote vmware host
+description:
+    - Gathers facts about remote vmware host.
+version_added: 2.5
+author:
+    - Wei Gao (@woshihaoren)
+requirements:
+    - python >= 2.6
+    - PyVmomi
+extends_documentation_fragment: vmware.documentation
+'''
+
+EXAMPLES = '''
+- name: Gather vmware host facts
+  vmware_host_facts:
+    hostname: esxi_ip_or_hostname
+    username: username
+    password: password
+  register: host_facts
+  delegate_to: localhost
+'''
+
+RETURN = '''
+instance:
```

Comment:
I didn't want you to remove everything, just the incorrect sample.

```yaml
instance:
  description: system info about the host machine		
  returned: always		
  type: dict		
```

Reply:
ok...